chunky/src/java/se/llbit/chunky/renderer/export/PictureExportFormat.java

```diff
@@ -50,12 +52,20 @@ default String getDescription() {
   String getExtension();
 
   /**
-   * Check if this format supports transparency (used for transparent sky).
+   * Note: It depends on the scene settings if the alpha buffer will be available on export.
    *
-   * @return True if this format supports transparency, false otherwise
+   * @return the required format for the alpha buffer or {@link AlphaBuffer.Type#UNSUPPORTED} if alpha is not supported.
+   */
+  default AlphaBuffer.Type transparencyType() {
+    return AlphaBuffer.Type.UNSUPPORTED;
+  }
+
+  /**
+   * @return true, if the export formats wants preprocessed buffer data<br>
+   *         false, if the export format uses the unprocessed sampling data
    */
-  default boolean isTransparencySupported() {
-    return false;
+  default boolean wantsPostprocessing() {
```

Comment:
```suggestion
  default boolean isPostprocessed() {
```

Reply:
This would break plugin compatability

Reply:
Also I like this name because it describes if the format expects a buffer with postprocessed data before exporting. See `Scene::writeFrame()`

Reply:
Oh wait, we had `wantsPostprocessing` before? :grimacing: Weird name, but allright.